magenta/models/nsynth/wavenet/fastgen.py

```diff
@@ -159,10 +165,10 @@ def save_batch(batch_audio, batch_save_paths):
     wavfile.write(name, 16000, audio)
 
 
-def synthesize(encodings,
+def synthesize(FLAGS, encodings,
```

Comment:
These functions are also called outside the main binary (for example, as imports in the colab notebooks). Could you refactor to make FLAGS an optional keyword so the functions can still be called as previously?